assets/tests/components/ladderPage.test.tsx

```diff
@@ -415,65 +407,16 @@ describe("LadderPage", () => {
     ;(useTimepoints as jest.Mock).mockImplementationOnce(
       () => timepointsByRouteId
     )
-    const tree = renderer
-      .create(
-        <StateDispatchProvider state={mockState} dispatch={mockDispatch}>
-          <BrowserRouter>
-            <RoutesProvider routes={routes}>
-              <LadderPage />
-            </RoutesProvider>
-          </BrowserRouter>
-        </StateDispatchProvider>
-      )
-      .toJSON()
-    expect(tree).toMatchSnapshot()
-  })
-
-  test("renders with vehicles and selected vehicles", () => {
```

Comment:
Note: I've removed this (and I think a couple of other places that test rendering selected vehicles) because #1748 moved the rendering of the VPP outside of the page component.